integrations/dalle/src/types.ts

```diff
@@ -0,0 +1,17 @@
+import * as botpress from '.botpress'
+import { z } from "zod";
+
+// Types
+export type TGenerateImageOutput = botpress.actions.generateImage.output.Output
+export type TDalleData = botpress.actions.generateImage.input.Input
+export type TContext = botpress.configuration.Configuration
+
+// Enums
+export const SizeEnum = z.enum(['1024x1024', '1792x1024', '1024x1792']).describe('Size of the image to generate.')
```

Comment:
**chore:** These zod schemas are only used to set constant values to the `buildApiData()` function. Use plain old constant instead like:

```ts
const defaultSize = "1024x1024"
const defaultQuality = "standard"
const defaultModel = "dall-e-3"
```